addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -411,6 +411,9 @@ export class HtmlField extends Component {
         this.Wysiwyg = wysiwygModule.Wysiwyg;
     }
     _isDirty() {
+        if (!this._getCodeViewEl() && !this.wysiwyg?.dirty) {
```

Comment:
Where does that `wysiwyg.dirty` come from? How come we weren't checking for it before?

Reply:
My changes did not went through, I re-pushforced to change `dirty` with `didUserMakeChanges`.